client/reader/recent/index.tsx

```diff
@@ -18,31 +20,31 @@ import './style.scss';
 const Recent = () => {
 	const dispatch = useDispatch< ThunkDispatch< AppState, void, AnyAction > >();
 	const [ selectedItem, setSelectedItem ] = useState< ReaderPost | null >( null );
+	const isWide = useBreakpoint( WIDE_BREAKPOINT );
 
 	const [ view, setView ] = useState< View >( {
 		type: 'table',
 		fields: [ 'seen', 'post' ],
 	} );
 
-	const { data, posts } = useSelector( ( state: AppState ) => {
-		const streamData = state.reader?.streams?.recent;
-		const postsMap: Record< string, PostItem > = {};
+	const data = useSelector( ( state: AppState ) => state.reader?.streams?.recent );
 
-		// Create a map of posts for all items
-		streamData?.items?.forEach( ( item: ReaderPost ) => {
+	const posts = useSelector( ( state: AppState ) => {
```

Comment:
I separated and refactored the selectors to prevent a console warning about them being recreated too often.